packages/svelte2tsx/src/svelte2tsx.ts

```diff
@@ -53,12 +58,20 @@ type pendingStoreResolution<T> = {
     scope: Scope;
 };
 
+/**
+ * Add this tag to a HTML comment in a Svelte component and its contents will
+ * be added as a docstring in the resulting JSX for the component class.
+ */
+const COMPONENT_DOCUMENTATION_HTML_COMMENT_TAG = '@component' as const;
```

Comment:
Is it necessary to have const assertions here?

Reply:
My intention was to make it read-only. Maybe it wasn't nexessary, but I don't think it hurts either. WDYT?

Reply:
I would remove it. A const is already `readonly` by definition and the type is also infered as `@component` I think.

Reply:
I'm learning this after the fact:

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions

What this `as const` would prevent is unwanted type widening. I think in this file it doesn't really matter, and if it's causing confusion, I would rather remove it.

Reply:
Note that in they docs they use `let bla = 'somestring'`. For let it is indeed typed as `string`, because you could change it, but `const` cannot be reassigned so TS should infer the exact string as a type.

Reply:
I removed the superfluous assertion now.